lib/resources/r4/patient_patch.yaml

```diff
@@ -0,0 +1,760 @@
+---
+name: Patient
+field_name_base_url: https://hl7.org/fhir/R4/patient-definitions.html#Patient
+operations:
+- name: add-identifier
+  path: /identifier/-
+  operation: add
+  cardinality: 1..1
+  type: Identifier
+  url: https://hl7.org/fhir/R4/patient-definitions.html#Patient.identifier
+  description: Add a new identifier for the patient.
+  example: |
+    [
+      {
+        "path": "/identifier/-",
+        "op": "add",
+        "value": {
+          "type": {
+            "coding": [
+              {
+                "code": "MR",
+                "system": "http://hl7.org/fhir/v2/0203"
+              }
+            ]
+          },
+          "system": "urn:oid:1.1.1.1.1.1",
+          "value": "&lt;unique identifier&gt;",
+          "period": {
+            "start": "2016-01-02T00:00:00-05:00",
+            "end": "2020-01-02T00:00:00-05:00"
+          }
+        }
+      }
+    ]
+  note: >
+    <ul>
+      <li>Must not contain the `assigner` and `use` fields.</li>
+      <li>Must contain the `type`, `system` and `value` fields, and must include a time component with a timezone when specifying a `period`.</li>
+    </ul>
+- name: test-identifier-id
+  path: /identifier/{index}/id
+  operation: test
+  cardinality: 1..1
+  type: string
+  url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
+  description: Test the value of the `Identifier.id` of the identifier at the given <code>{index}</code> in the list of patient identifiers.
+  example: |
+    [
+      {
+        "path": "/identifier/1/id",
+        "op": "test",
+        "value": "CI-9926250-1"
+      }
+    ]
+- name: remove-identifier
+  path: /identifier/{index}
+  operation: remove
```

Comment:
cardinality and type?

couple others don't have these keys either

Reply:
So for remove operations a value is not required. Made sense to not set the cardinality/type instead of setting it to `N/A`

Reply:
having blank boxes looks to me like we forgot something, I would prefer an N/A

Reply:
Updated cardinality to "0" and type to "N/A"
https://github.com/cerner/fhir.cerner.com/pull/242/commits/9a75bb6a3e406c75368166a12cb2d3fa039374ca